proxies/live/apiproxy/policies/OAuthV2.GenerateAccessToken.xml

```diff
@@ -13,6 +13,8 @@
       <Attribute name="id_token" ref="externalIdToken" display="true"/>
       <Attribute name="id_token-subject" ref="jwt.VerifyJWT.FromExternalIdToken.claim.subject" display="true"/>
       <Attribute name="id_token-issuer" ref="jwt.VerifyJWT.FromExternalIdToken.claim.issuer" display="true"/>
+      <Attribute name="id_token-acr" ref="jwt.VerifyJWT.FromExternalIdToken.claim.acr" display="true"/>
```

Comment:
Are these important enough that we should expose them as additional attributes? We could just make people decode the id token if they need to access them

Reply:
I think the question I'm really asking is: Will these only ever be used in the Signing Service API?

Reply:
how exactly are these being used in the signing service? i'm in favour of having the consumer have to decode the token if these claims aren't widely / frequently used by other APIs

Reply:
The answer at the moment is they're not being used at all.
Agreed. I think we need Rich to weigh in on this.

Reply:
At least two APIs will use these attributes (EPS & Signing Service), and Rich suspects more will use them in future, so maybe worth exposing them?

Reply:
sure, sounds like they wont be dead weight for the most part :+1: